### PR TITLE
Add RPG_RT compatible scene delays

### PR DESCRIPTION
**Pull request by fmatthew5876**
* New Game -> 60 frames
* Load Game -> 60 frames
* Return to Title -> 20 frames

Fix: #1939

### CODE REVIEW

**Comment by fmatthew5876** [comment]
@Br4ssman Could you please confirm this fixes the issues in your game

**Comment by Br4ssman** [comment]
It works, 100/10.

File: src/scene.cpp [hunk 1]
```diff
@@ -34,6 +34,8 @@
 #endif
 
 
+constexpr int Scene::kStartGameDelayFrames;
+constexpr int Scene::kReturnTitleDelayFrames;
```
Comment (carstene1ns) [inline src/scene.cpp]: Why are these needed? These should get pulled in from the header.

Comment (fmatthew5876) [reply src/scene.cpp]: static data members need definitions. Without them you'll likely get linker errors in debug builds as unoptimized code is trying to find the address of these constants.
